### ISSUE TITLE
Mounted Horse disappears when logging in 

### ISSUE THREAD

**Issue by EltoCode** [AuthMe/AuthMeReloaded#2042]
### What behaviour is observed:
When teleport unregistered players to spawn is set to true, If a player logs out while riding a horse, and the session expires, The horse is deleted when logging back in

### What behaviour is expected:
The Horse should not be deleted

### Steps/models to reproduce:
login
Mount a horse
Logout while still mounted (make sure the session is disabled or expires)
Login

### Plugin list:
This can be found by running `/pl`

### Environment description
Standalone server/, MySql, ...

### AuthMe build number:
5.6.0

**Comment by Pawialdo** [comment]
Happens on my server. Im using multiverse + multiverse inventories, world border. Worlds "spawn" and "survival" have separated inventories.
1. Player logs out while mounted on a horse/donkey on "survival" world with coordinates {x,y,z}
2. Connects again to the server on "spawn" world - in location where registered players type /login
3. After logging in Player is teleported to the same {x,y,z} coordinates on the "spawn" world. With the horse/donkey even though i have world border set
4. Player gets instantly teleported to the world he logged out of (in this case "survival"). The horse/donkey is left behind in the "spawn" world. Player ususally cannot move, or suffocates inside walls. He looks as if he was standing, without the horse, 
5. After Player leaves and rejoins server he can move. Paper removes duplicate entity with the same UUID as if the horse was in both worlds? I dont know really.

So basically player either dies or cannot move and after relogin his horse/ donkey with items in chest is either stuck in spawn world, or gone completely.

**Comment by QuanTrieuPCYT** [comment]
same issue here on my SMP server
currently looking forward to a fix :eyes:
